[2024-07-07T15:04:49.122+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-07T15:04:49.632+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_V2.third_task scheduled__2024-07-02T00:00:00+00:00 [queued]>
[2024-07-07T15:04:49.899+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_V2.third_task scheduled__2024-07-02T00:00:00+00:00 [queued]>
[2024-07-07T15:04:49.918+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 6
[2024-07-07T15:04:50.332+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): third_task> on 2024-07-02 00:00:00+00:00
[2024-07-07T15:04:50.575+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_V2', 'third_task', 'scheduled__2024-07-02T00:00:00+00:00', '--job-id', '106', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp3cstcyhe']
[2024-07-07T15:04:50.654+0000] {standard_task_runner.py:91} INFO - Job 106: Subtask third_task
[2024-07-07T15:04:50.859+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=218) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-07T15:04:50.876+0000] {standard_task_runner.py:63} INFO - Started process 238 to run task
[2024-07-07T15:04:51.778+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag_V2.third_task scheduled__2024-07-02T00:00:00+00:00 [running]> on host a10fdcb539f4
[2024-07-07T15:04:54.767+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='bavenraj' AIRFLOW_CTX_DAG_ID='our_first_dag_V2' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T00:00:00+00:00'
[2024-07-07T15:04:54.772+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-07T15:04:56.324+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-07T15:04:56.371+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task 3 and will be running after task2 !']
[2024-07-07T15:04:56.571+0000] {subprocess.py:86} INFO - Output:
[2024-07-07T15:04:56.616+0000] {subprocess.py:93} INFO - hey, I am task 3 and will be running after task2 !
[2024-07-07T15:04:56.629+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-07T15:04:56.637+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-07T15:04:56.813+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_first_dag_V2, task_id=third_task, run_id=scheduled__2024-07-02T00:00:00+00:00, execution_date=20240702T000000, start_date=20240707T150449, end_date=20240707T150456
[2024-07-07T15:04:57.202+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-07T15:04:57.404+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-07T15:04:57.435+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
